lib/rom/repository/changeset/pipe.rb

```diff
@@ -5,6 +5,7 @@ module ROM
   class Changeset
     # Built-in ROM::Repository transformations available Changeset
     #
+    # @todo merge into PipeRegistry when transproc <= 0.4.2 support is dropped
```

Comment:
Can we do that already? If it can simplify rom-repo then I'm all for it. I should be able to wrap up transproc 1.0.0 this way